test/lit/passes/memory-packing_zero-filled-memory64.wast

```diff
@@ -0,0 +1,342 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+
+;; RUN: wasm-opt %s --memory-packing -all --zero-filled-memory --enable-memory64 -S -o - | filecheck %s
+
+(module
```

Comment:
Is this file's input copied from an existing test? If so, it would be better to add a line ending in `filecheck %s --check-prefix WASM64`, in addition to the normal line ending in `filecheck %s`. Then the same input will be run in both modes in the same file.

I was thinking that a small new file would be good enough. But actually, full coverage in the main existing file might be better, to cover more things, with a second filecheck line.